packages/apps/kadena-transfer/.eslintrc.js

```diff
@@ -0,0 +1,78 @@
+require('@rushstack/eslint-config/patch/modern-module-resolution');
+const fs = require('fs');
+
+const prettierOptions = JSON.parse(fs.readFileSync('./.prettierrc', 'utf8'));
+
```

Comment:
Can we remove everything from here and use the default config? You can use this one as example https://github.com/kadena-community/kadena.js/blob/17792c8301c9ba2f4e436b36483a0cc90abaa3c3/packages/apps/kadena-docs/.eslintrc.cjs